Views/ResponsiveMediaField.cshtml

```diff
@@ -1,9 +1,13 @@
 ﻿@model Etch.OrchardCore.Fields.ResponsiveMedia.ViewModels.DisplayResponsiveMediaFieldViewModel
 
+@{
+    var cssClass = string.Format("{0}__media--{1}", Model.PartFieldDefinition.PartDefinition.Name, Model.PartFieldDefinition.Name);
```

Comment:
I'd suggest using `$"{Model.PartFieldDefinition.PartDefinition.Name}__media--{Model.PartFieldDefinition.Name}" instead of `string.Format`.